Sources/CorePayments/Networking/APIClient.swift

```diff
@@ -51,12 +52,26 @@ public class APIClient {
         return try await http.performRequest(httpRequest)
     }
     
-    // TODO: - Add GraphQL equivalent request type & function
-    // public func fetch(request: GraphQLRequest) async throws -> HTTPResponse { }
+    /// :nodoc:
+    public func fetch(request: GraphQLRequest) async throws -> HTTPResponse {
+        let url = try constructGraphQLURL(queryName: request.queryNameForURL)
+
+        let postBody = GraphQLHTTPPostBody(query: request.query, variables: request.variables)
+        let postData = try JSONEncoder().encode(postBody)
+                
+        let httpRequest = HTTPRequest(
+            headers: [.contentType: "application/json"],
+            method: .post,
+            url: url,
+            body: postData
+        )
+        
+        return try await http.performRequest(httpRequest)
+    }
     
     // MARK: - Private Methods
     
-    private func constructURL(path: String, queryParameters: [String: String]) throws -> URL {
+    private func constructRestURL(path: String, queryParameters: [String: String]) throws -> URL {
```

Comment:
I think REST is typically all caps right?

Reply:
It is, I had `constructRESTURL()` before, but that looked odd to me. Which do you think is better?

Reply:
I think we have something similar with `constructGraphQLURL` as well. Agree both look a little awkward but not sure if there is a way around it. 🤷 It's private so not a super strong preference.

Reply:
Ok, I updated to `constructRESTURL()` d3fe18806d8df1cd523662e998ecb2ff46263ba3